### DISCUSSION TITLE
JSR

### DISCUSSION THREAD

**Discussion by ukqfhf** [VKCOM/VKUI#6942] [Ideas] [answered]
Недавно в открытую бету вышел [JSR](https://jsr.io/) open-source package registry.
Заявлено:
- Native TypeScript support
- ECMAScript modules only
- Cross-runtime support
- JSR is a superset of npm
- Outstanding developer experience
- Fast, secure, and reliable

Из [презентации](https://www.youtube.com/watch?v=MFCn4ce5dVc) складывается ощущение что технология действительно интересная и решает множество проблем. 
Есть ли планы опубликовать vkui?

**Comment by SevereCloud** [comment, marked as answer]
Привет, JSR предназначен для правильной работы библиотеки в разных средах выполнения JS(например Node.js, Deno, Bun, Cloudflare Workers). 

VKUI это библиотека предназначенная для веб сборщиков(Webpack, Vite, Next JS и т.д.). Для нормальной работы в старых браузерах, необходимо поставлять библиотеку в определенной спецификации(в нашем случае es2015)
